src/main/java/io/camunda/testing/filters/StreamFilter.java

```diff
@@ -0,0 +1,18 @@
+package io.camunda.testing.filters;
+
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.record.value.ProcessInstanceRecordValue;
+import io.camunda.zeebe.protocol.record.value.ProcessMessageSubscriptionRecordValue;
+
+public class StreamFilter {
+
+  public static ProcessInstanceRecordStreamFilter processInstance(
```

Comment:
:wrench: I think we could add convenience methods that take `RecordStreamSource`

`StreamFilter.processInstance(recordStreamSource.processInstanceRecords())`-> `StreamFilter.processInstance(recordStreamSource))`